docs/tremor-query/index.md

```diff
@@ -113,14 +113,49 @@ A tumbling window is a window configured with a fixed non-overlapping interval o
 
 Support for sliding windows has not been implemented yet (it has an [open RFC](https://rfcs.tremor.rs/0004-sliding-window-mechanism/) and it will be picked up for a future release).
 
+### Tumbling Windows
+
+Tremor supports tumbling windows by number of events or by time.
+
+General configuration Parameters:
+
+* `eviction_period`: duration in nanoseconds without events arriving, after which to close an existing window, if it is full or not.
+
+#### Windows based on number of events
+
+Size based tumbling windows close when a certain number of aggregated events has been reached.
+
+Configuration Parameters:
+
+- `size`: Number of events until this window closes and emits a downstream event (The ).
+
+The size increment for each event defaults to `1` but can be customized by the embedded script in the window definition.
+This script needs to return an unsigned integer denoting the number of events to use for this event.
+It is possible to ignore the current event by emitting `0`.
+
+#### Windows based on time
+
+Time based tumbling windows close when a certain duration has been elapsed. The source for measuring the duration
+is the `ingest` timestamp of the events flowing through by default. The provided embedded script can be used to customize the
+source of time measurement. The embedded script must return a number representing a timestamp in nanoseconds. 
+This way windows using other timestamps than event ingest time can be built.
```

Comment:
than -> that

Reply:
really? `other timestamps that event ingest time`?

Reply:
`than` reads better to me too, I never heard `other that`

perhaps rephrasingf would make ti cleaner?
```
This way windows using timestamps other than the event ingest time can be built.
```